src/transformer/before/visitor/visit-call-expression.ts

```diff
@@ -44,7 +44,7 @@ export function visitCallExpression(
               factory.createPropertyAssignment(
                 "identifier",
                 factory.createStringLiteral(
-                  node.typeArguments[0].getFullText().trim()
+                  node.typeArguments[0].getFirstToken()!.getFullText().trim()
```

Comment:
That NonNullAssertion there feels a bit unsafe. I would prefer that we handle it a bit more gracefully, ideally by taking the full text of the first token if it present, and falling back to taking the full text of the first type argument otherwise. 
Something like:

```ts
// Move it up in a constant
const [firstTypeArgument] = node.typeArguments;

return factory.updateCallExpression(
          node,
          node.expression,
          node.typeArguments,
          [
            factory.createObjectLiteralExpression([
              factory.createPropertyAssignment(
                "identifier",
                factory.createStringLiteral(
                  (firstTypeArgument.getFirstToken()?.getFullText() ?? firstTypeArgument.getFullText()).trim()
                )
              ),
            ]),
          ]
        );
```